src/style.css

```diff
@@ -0,0 +1,126 @@
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 600;
+}
+
+html {
+  display: block;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+}
+
+.search-form {
```

Comment:
Some of the rules are unnecessary or repeated, there is no need to have a class differentiation between big and small form, you can have a regular default `search-form` and `search-form--small` that modifies some of the values
```css
.search-form {
  display: flex;
  justify-content: center;
  margin-top: 20px;
  width: 100%;
  height: 70px;
}

.search-form--small {
  height: 42px;
}
```

Reply:
Same for the rest of the classes e.g.
```css
.search-form_label {
  position: relative;
  display: block; -> are you sure this is needed?
  width: 100%;
}

.search-form__label:after {
  content: "";
  display: block;  -> same here, if not remove it
  position: absolute;
  background-image: url("./images/Search.svg");
  background-repeat: no-repeat;
  background-size: contain;
  width: 19px;
  height: 19px;
  top: 25px;
  left: 26px;
}

.search-form__label--small:after {
  width: 11px;
  height: 11px;
  top: 15px;
  left: 13px;
}
```

Reply:
- [DEMO LINK](https://KarolinaWiktoria.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://KarolinaWiktoria.github.io/layout_search-bar-airbnb/report/html_report/)

once again :)